designer/client/src/types/node.ts

```diff
@@ -53,7 +53,7 @@ export type Field = Parameter | FragmentInputParameter;
 export interface Parameter {
     uuid: string;
     name: string;
-    expression: Expression;
+    expression?: Expression; //TODO: Check if we need this type
```

Comment:
In what situations is this null?

Reply:
I think it's something redundant, I checked and there is no use in removing it. Reverted ✔️